### ISSUE TITLE
Set allow_resize_to_same_host as Overridable in nova.conf

### ISSUE THREAD

**Issue by seancarlisle** [rcbops/chef-cookbooks#923]
The default value for allow_resize_to_same_host in /etc/nova/nova.conf is false and it is currently not overrideable in the cookbooks.  Customers are requesting this to be set to true to prevent instances from unnecessarily migrating to other hypervisors due to a resize.

**Comment by breu** [comment]
At current this flag only allows the same host as a target for the migration.  This flag does not force the migration to use the same host so based on the utilization if the compute hosts there still exists a high probability that the migration will still move the instance to a new host.

**Comment by breu** [comment]
That being said - I am all for adding this tunable.

**Comment by seancarlisle** [comment]
I agree that it doesn't.  However, this flag is very useful in resource-constrained environments where the current host has sufficient resources to accept the resize, but any other host does not.  With the flag set to false, the resize would fail since no other host would be able to accept the resized instance.

**Comment by slashterix** [comment]
+1
